app/components/stepper-signup.js

```diff
@@ -19,4 +24,18 @@ export default class StepperSignupComponent extends Component {
   @action handleInputChange(key, value) {
     set(this.signupDetails, key, value);
   }
+
+  @action incrementStep() {
+    if (this.currentStep < 5) {
+      this.currentStep += 1;
+      const queryParams = { dev: true, step: 1 };
+      localStorage.setItem('currentStep', this.currentStep);
```

Comment:
as we discuss we donot store currentstep on localstorage in signup form

there is other way you can do this:- refer (https://github.com/Real-Dev-Squad/website-my/blob/develop/app/templates/new-signup.hbs#L3, https://github.com/Real-Dev-Squad/website-my/blob/develop/app/controllers/new-signup.js#L40)